kingfisher_scrapy/spiders/honduras_iaip.py

```diff
@@ -0,0 +1,55 @@
+import scrapy
+
+from kingfisher_scrapy.base_spider import SimpleSpider
+from kingfisher_scrapy.exceptions import SpiderArgumentError
+from kingfisher_scrapy.util import components, handle_http_error
+
+
+class HondurasIAIP(SimpleSpider):
+    """
+    Domain
+      Instituto de Acceso a la Información Publica (IAIP)
+    Spider arguments
+      portal
+        Filter by portal:
+
+        covid19
+          Emergencia Covid-19
```

Comment:
```suggestion
          IAIP Emergencia Covid-19
```

as in https://portalunico.iaip.gob.hn/datosabierto/

Reply:
done